lib/main.dart

```diff
@@ -33,19 +33,18 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'Sufficient Goldfish',
       theme: ThemeData.light(), // switch to ThemeData.day() when available
-      home: FishPage(PageType.shopping, deviceId),
+      home: FishPage(deviceId),
       debugShowCheckedModeBanner: false,
     );
   }
 }
 
-enum PageType { shopping, reserved }
+enum ListType { available, reserved }
```

Comment:
nit: I kinda liked the name PageType because it was a little clearer since there tend to be a lot of lists in programming generally. What if instead we called it "ViewType"?

Reply:
SGTM, changed to ViewType.